ingeniamotion/fsoe.py

```diff
@@ -394,6 +394,7 @@ def __init__(self, motion_controller: "MotionController") -> None:
         self.__handlers: Dict[str, FSoEMasterHandler] = {}
         self.__next_connection_id = 1
         self._error_observers: List[Callable[[FSoEError], None]] = []
+        self.__fsoe_configured = False
```

Comment:
I think this flag should go to the respective FSoEMasterHandler.

Potentially you could do:
- create fsoe handler 1
- start fsoe handler 1
- create fsoe handler 2
- Stop all handlers. Fsoe handle 2 will be commanded to stop while it is already stopped.

Maybe with the current api it's not possible to do this but for consistency sake I would keep track of each handler

Reply:
I keep using this property, but now I check if the master handler is running 65a7f1e